examples/emr/v1beta1/instancefleet.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: emr.aws.upbound.io/v1beta1
+kind: InstanceFleet
+metadata:
+  annotations:
+    meta.upbound.io/example-id: emr/v1beta1/instancefleet
```

Comment:
If an emr cluster resource is required to create this resource, we should add a manual intervention specifying this, or we should add a working emr cluster resource and the necessary resources to this YAML file.

Reply:
Yes I implemented the manual intervention. Thank you!